package.json

```diff
@@ -18,12 +18,11 @@
   "author": "Me",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.21.1",
```

Comment:
Je connais juste de nom axios mais ça apporte quoi de plus par rapport à fetch par exemple ?

Reply:
This is this lib : https://www.npmjs.com/package/axios

It is mostly for convenience. `fetch` won't reject the Promise if the HTTP status code is other than `200` and it has those `get`/`post` ready to use calls. But we might rethink it. Using `fetch` could be as easy as axios and reduce the number of dependencies.